example/example18.cpp

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Sry, I am bad at using cmake. When I add example18 to PYBIND11_EXAMPLES I'll get a library, but I need an executable. I don't know how that should work.